src/views/pages/apply.ejs

```diff
@@ -3,6 +3,8 @@
 
```

Comment:
Would be great to somehow indicate which fields are required but I'm guessing that is not a trivial change since the form is generated dynamically

Reply:
I am working on this, just thinking about how to design it. The questions all have a setting called `required` so it shouldn’t be too difficult.

Right now the required inputs turn red when you click and purple otherwise but it should be made clearer